### ISSUE TITLE
performance on pipal_val21

### ISSUE THREAD

**Issue by linkboyx** [IIGROUP/MANIQA#8]
Hello,  would you please release the performance on pipal_val21? I dont find that in your paper.
I want to know that because in my experiment, using pipal_val21 and pipal_train21, it seems that the model overfitting quickly.
The best val SRCC/PLCC (best about ~0.64) appears in the the first serveral epochs, and decreases persistently (down to ~0.4) while the metric in training always increases.
I tried to increase the dropout rate, but no obvious effect.
Should I stop the training earlier? Any suggestions are appreciated.

**Comment by TianheWu** [comment]
Hi!
Thanks for your questions!

We found the problem of overfitting when experimenting on both PIPAL_21 and PIPAL_22. The sponsor also found that there is an overfitting problem in training on the PIPAL dataset. Therefore, we need to find new ways to solve it. 

<img width="774" alt="f44044dd3fe114a35a8339ccbb33424" src="https://user-images.githubusercontent.com/63295426/169453048-6c35c60a-e950-41b1-b883-4d703e870bf9.png">

We can achieve the effect of the above figure by training on PIPAL21. The epoch one is the best performance. In the future, we will also continue to solve the problem of overfitting. 

Maybe, you need to initialize the model parameters can make the performace better? Or, pay attention to the environments.

**Comment by linkboyx** [comment]
I see.  Maybe the dataset is too small for modern networks.
Thank you for your sharing. :)
